src/gui/qgsmaplayerstylecategoriesmodel.cpp

```diff
@@ -295,7 +295,7 @@ QVariant QgsMapLayerStyleCategoriesModel::data( const QModelIndex &index, int ro
         case Qt::ToolTipRole:
           return tr( "Notes" );
         case Qt::DecorationRole:
-          return QIcon(); // TODO
+          return QgsApplication::getThemeIcon( QStringLiteral( "/mIndicatorNotes.svg" ) ); // TODO
```

Comment:
@alexbruy , I agree the // TODO shouldn't be removed :wink: -- 

I'm -1 to using the outline-only notes indicator icon. As the screenshot in the description shows, it doesn't sit nicely alongside the other icons in there, and due to the SVG icon size difference, the outlines in the note icon appears "oversized". 

For now, maybe we can add the elevation scale icon and keep the notes icon missing for now?

Reply:
I do not have a strong opinion here, if you think it does not fit here let's remove it and keep only elevation icon.